includes/core/classes/class-block.php

```diff
@@ -72,4 +73,73 @@ public function register_blocks(): void {
 			);
 		}
 	}
+
+
+	/**
+	 * Register block patterns.
+	 *
+	 * This method registers multiple different block-patterns for GatherPress.
+	 *
+	 * @since 1.0.0
+	 * @see   https://developer.wordpress.org/reference/functions/register_block_pattern/
+	 *
+	 * @return void
+	 */
+	public function register_block_patterns(): void {
+
+		/**
+		 * Made to be used with the 'template' parameter
+		 * when registering the 'gatherpress_event' post type
+		 * and will not be visible to the editor at any point.
+		 */
+		\register_block_pattern(
+			'gatherpress/event-template',
+			array(
+				'title'    => 'Invisible Event Template Block Pattern',
```

Comment:
Needs internationalization __().

Reply:
This is a little funny.

1. I had _() everywhere in place
2. Than realised, that nobody will ever see this. Because it's hidden. So I removed the _() and added a comment "// Needs no i18n
3. Lastly I removed the comment also, because .... I don't know.

Now I'm going to add _() again, yes.